python-sdk/src/astro/dataframes/load_options.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from attr import define
+from pandas._typing import DtypeArg
+
+from astro.options import LoadOptions
+
+
+@define
+class PandasLoadOptions(LoadOptions):
+    pass
+
+
+@define
+class CsvLoadOption(PandasLoadOptions):
```

Comment:
I believe it is very misleading to have `CsvLoadOption` which is specific to Pandas. I think, for now, we should focus on having database/pandas-specific Options classes.

Reply:
@tatiana maybe we have a `LoadFormat` class instead as a class member of the `PandasLoadOptions` class?

Reply:
@tatiana for pandas options we call different methods of pandas lib based on the file type like `read_csv`, `read_parquet` etc and these methods accept different param. I added `CsvLoadOptions` because it would be easy to understand for a user to understand what they can pass if they are loading CSV file using pandas also if we will have a class instead of dict editor will also provide suggestions. But as @utkarsharma2 rightly pointed out the origin of this chnage is https://github.com/astronomer/astro-sdk/pull/1466 so we can take this discussion there.